loop/storage/redis.js

```diff
@@ -3,7 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 "use strict";
-var redis = require("redis");
+var redis = require("./redis_client");
```

Comment:
Also, tweaking imports depending on a configuration is a known bad practice because it makes things harder to test.